README.markdown

```diff
@@ -22,3 +22,51 @@ long the outcome is distributed under the same license. Otherwise, you must
 obtain a [commercial license](./LICENSE-COMMERCIAL) that removes such
 restrictions. Read more about our licensing approach
 [here](https://www.sourcemeta.com/licensing/).
+
+Example
+--------------
+
+```cpp
+#include <sourcemeta/jsontoolkit/json.h>
+#include <sourcemeta/jsontoolkit/jsonpointer.h>
+#include <sstream>
+#include <iostream>
+
+int main()
+{
+    // creating json using parse
+    sourcemeta::jsontoolkit::JSON _template_ =
+        sourcemeta::jsontoolkit::parse(R"JSON({
+            "name" : "xyz" ,
+            "age" : 20 ,
+            "address" : "zxy"
+        })JSON");
+
+    // stringstream for output
+    std::ostringstream stringvar;
+    // pointer for the name
```

Comment:
Again, be mindful with comments. "pointer for the name" doesn't mean much, plus you not only have JSON Pointers for the `name` property (so the comment is actually incomplete). A better comment would have been "Instantiate a JSON Pointer for the `name` property"